docs/PARAMETERS.md

```diff
@@ -36,11 +36,11 @@ Transmission power in Watts.
 
 #### Tx Diversity #### 
 Diversity mode. 
-Can be "enabled", "antenna1", "antenna2". 
+Can be "enabled", "antenna1", "antenna2", "r:en, t:ant1", "r:en, t:ant2".
 
 #### Tx Ch Source #### 
 Selects the source from which the rc data should be read. 
-Can be "none", "mbridge", "in", "crsf".
+Can be "none", "mbridge", "in", "crsf", "elrs rx".
```

Comment:
this option is not going to come
it was for a special version for one potential user, but - AFAIK - has not even been tried

what has changed is the sequence of options, if I recall correctly mbridge and crsf were swapped :)

Reply:
Updated

Reply:
should be  "none", "crsf", "in", "mbridge" I think, https://github.com/olliw42/mLRS/blob/main/mLRS/Common/setup_list.h#L95
:)